src/microns_to_nwb/convert_microns_to_nwb.py

```diff
@@ -0,0 +1,15 @@
+from tools.behavior import add_eye_tracking, add_treadmill
+from tools.intervals import add_trials
+from tools.nwb_helpers import start_nwb
+from tools.ophys import add_ophys
+
+
+def build_nwb(scan_key):
+    nwb = start_nwb(scan_key)
+    # add_stimulus(scan_key, nwb)
```

Comment:
What's the status of stimulus?

Reply:
It works, it was commented out in the notebook because of RAM issues which were solved since then.